### PR TITLE
7 wider graphic

### PR DESCRIPTION
**Pull request by marianmosley**

### CODE REVIEW

**Comment by mtc2013** [comment]
@tansaku what say you about this potential fix?

**Comment by mtc2013** [comment]
We should probably also bump the gem version number ever so slightly.

File: lib/project_metric_slack.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 require 'slack'
 require 'color_functions'
+require 'byebug'
```
Comment (mtc2013) [inline lib/project_metric_slack.rb]: This needs to be removed since it doesn't exist in production.

**Comment by tansaku** [comment]
@marianmosley @mtc2013 looks good to me - will see if I can't get it onto staging today

**Comment by tansaku** [comment]
@mtc2013 the interesting thing is that we can deploy without bumping the gem version - `bundle update` will grab the latest based on commit id - not saying that's ideal, but interesting to know we can ...

**Pull request merged**